packages/teams-js/src/internal/constants.ts

```diff
@@ -136,6 +136,7 @@ export const validOrigins = [
   'outlook-sdf.office.com',
   'outlook.office365.com',
   'outlook-sdf.office365.com',
+  'outlook.live.com',
```

Comment:
Do you have any test environment URLs you also need to match? For example, *.teams.microsoft.com below is used to match some of the teams test environments.